libraries/dawg-client/pom.xml

```diff
@@ -8,6 +8,7 @@
     </parent>
 
     <artifactId>dawg-client</artifactId>
+    <version>2.0.2-SNAPSHOT</version>
```

Comment:
I changed all the versions to be explicitly set rather than inferred. We use the maven set versions plugin in continuous deployment and it needs the versions to be explicitly set. These values of a version should be handled automatically like in the maven-release-plugin or maven set versions plugin anyway so it doesn't require them to all be changed one by one manually.